packages/components/src/Select/Select.test.tsx

```diff
@@ -1,67 +1,195 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { ThemeProvider } from '@marigold/system';
 import { Select } from './Select';
+import { Item } from 'react-stately';
 
 const theme = {
-  select: {
-    default: {
+  button: {
+    select: {
       fontFamily: 'Inter',
     },
-    other: {
-      fontFamily: 'Oswald',
-    },
   },
 };
 
-test('supports default variant and themeSection', () => {
+test('supports default variant', () => {
   render(
     <ThemeProvider theme={theme}>
-      <Select title="select">
-        <option>1</option>
+      <Select label="MyLabel" data-testid="selectId">
+        <Item>1</Item>
       </Select>
     </ThemeProvider>
   );
-  const select = screen.getByTitle(/select/);
-
+  const select = screen.getByTestId('selectId');
+  expect(select).toBeDefined();
   expect(select).toHaveStyle(`font-family: Inter`);
 });
 
-test('accepts other variant than default', () => {
+test('accepts custom styles prop className', () => {
+  render(
+    <ThemeProvider theme={theme}>
+      <Select
+        label="MyLabel"
+        className="custom-class-name"
+        data-testid="selectId"
+      >
+        <Item>1</Item>
+      </Select>
+    </ThemeProvider>
+  );
+  const select = screen.getByTestId('selectId');
+  expect(select.className).toMatch('custom-class-name');
+});
+
+test('supports label with htmlFor prop', () => {
+  render(
+    <ThemeProvider theme={theme}>
+      <Select label="MyLabel">
+        <Item>1</Item>
+      </Select>
+    </ThemeProvider>
+  );
+  const selectLabel = screen.getAllByText(/MyLabel/);
+  expect(selectLabel[0]).toHaveAttribute('for');
+});
+
+test('supports disabled prop', () => {
   render(
     <ThemeProvider theme={theme}>
-      <Select title="select" variant="other">
-        <option>1</option>
+      <Select label="MyLabel" data-testid="selectId" disabled>
+        <Item>1</Item>
       </Select>
     </ThemeProvider>
   );
-  const select = screen.getByTitle(/select/);
+  const select = screen.getByTestId('selectId');
+  expect(select).toHaveAttribute('disabled');
+});
 
-  expect(select).toHaveStyle(`font-family: Oswald`);
+test('supports placeholder prop', () => {
```

Comment:
What does the test do? The prop doesn't matter -> for the user: "You can set a placeholder"

Reply:
I thought we test this that nobody can remove the placeholder prop from our component. That custom placeholder works.